ViderumGlobal/ckanext-querytool
# Implement drag and drop for visualization items

### Proposed fixes
This PR implements drag and drop for visualization items as well as a bug is fixed for displaying correct values for the items.

### How to test
Create a query tool and add a few charts. Then try to reorder them and save them. Reload the page and the items should keep the order.

### Features

- [x] includes new features
- [x] includes bugfix
